Week1/solution1.py

```diff
@@ -0,0 +1,12 @@
+n=int(input("Enter number:"))
+if n>1:
+    for i in range(2,int(n/2)+1):
+        if n%i==0:
+            print("{} is not prime number".format(n))

```

Comment:
can you suggest me how to write this line using f string?

Reply:
i have no idea about this